src/convertToObject.js

```diff
@@ -3,16 +3,36 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes string with styles
- * and returns object where CSS properties are keys
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
+ * and returns an object where CSS properties are keys
  * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .split(';')
+    .filter(element => element.trim());
+
+  const stylesResult = styles.reduce((acc, item) => {
```

Comment:
Don't create a `stylesResult` variable. Chain `filter` and `reduce` methods instead.